json/testnet-bitcoin-config.json

```diff
@@ -1,13 +1,13 @@
 {
   "bitcoinFeeSpendingCutoffPeriodInBlocks": 1,
   "bitcoinFeeSpendingCutoff": 0.001,
-  "bitcoinPeerUri": "http://localhost:18332",
+  "bitcoinPeerUri": "http://bitcoin-node:18332",
```

Comment:
Note these changes are to favour the execution of the services in a docker environment too